MBBSEmu/CPU/CPUCore.cs

```diff
@@ -3437,6 +3663,54 @@ private void Op_Fchs()
             FpuStack[Registers.Fpu.GetStackTop()] = -FpuStack[Registers.Fpu.GetStackTop()];
         }
 
+        /// <summary>
+        ///     Extended Shift Left
+        /// </summary>
+        private void Op_Shld()
+        {
+            uint result = 0;
+            switch (_currentOperationSize)
+            {
+                case 4:
+                    result = Op_Shld_32();
+                    break;
+                default:
+                    break;
+            }
+
+            WriteToDestination(result);
+        }
+
+        private uint Op_Shld_32()
+        {
+            var destination = GetOperandValueUInt32(_currentInstruction.Op0Kind, EnumOperandType.Destination);
+            var source = GetOperandValueUInt32(_currentInstruction.Op1Kind, EnumOperandType.Source);
+            var count = GetOperandValueUInt8(_currentInstruction.Op2Kind, EnumOperandType.Count);
+
+            //Make sure Count isn't > 32 bits
+            if (count >= sizeof(uint) * 8)
```

Comment:
prior code had `count >` and this is `count >=`, just wanted to clarify if this is accurate

Reply:
Updated this to a modulo 32 as per the documentation. The previous IF was for the case listed where if the shift amount was `>=` the operand size.

![image](https://user-images.githubusercontent.com/1139047/103445086-f0b45c00-4c3d-11eb-871a-072363375e3e.png)